src/controller/python/chip-device-ctrl.py

```diff
@@ -743,9 +749,15 @@ def main():
             dest="bluetoothAdapter",
             default="hci0",
             type="str",
-            help="Controller bluetooth adapter ID",
+            help="Controller bluetooth adapter ID, use disable to disable bluetooth.",
```

Comment:
Did you mean "use --no-ble to disable bluetooth"?

Reply:
Yes, fixed now, thanksl.